common4j/src/main/com/microsoft/identity/common/java/flighting/CommonFlight.java

```diff
@@ -124,7 +124,12 @@ public enum CommonFlight implements IFlightConfig {
     /**
      * Flight to enable handling the UI in edge to edge mode
      */
-    ENABLE_HANDLING_FOR_EDGE_TO_EDGE("EnableHandlingEdgeToEdge", true);
+    ENABLE_HANDLING_FOR_EDGE_TO_EDGE("EnableHandlingEdgeToEdge", true),
+
+    /**
+     * Flight to enable the new Android wrapped key loader.
+     */
+    ENABLE_NEW_ANDROID_WRAPPED_KEY_LOADER("EnableNewAndroidWrappedKeyLoader", true);
```

Comment:
Do you need this in this PR?

You could add this in next PR and factory class

Reply:
move for next pr